packages/git/README.md

```diff
@@ -26,12 +26,12 @@ See the "Troubleshooting" section of this README if you encounter any issues ins
 ## Configuration
 
 Data plugins are configured in `cardhost/data-sources`.
-The only required configuration is to set either the `repo` or `remote`.
+The only required configuration is to set either `repo` or `remote`.
 
 | Parameter | Type |  Description |
 |-----------|------|-------------|
 | `remote` | object |  Use a remote git repository, such as one hosted on Github. The object should contain a `url` string and the absolute path to your `privateKey` as a string. Example: `remote: { url: 'some-url', privateKey: process.env.GIT_PRIVATE_KEY}`|
-| `repo` | string | The path to a local git repository that already exists on disk |
+| `repo` | string | The path to a local git repository that already exists on disk, and has an initial commit |
```

Comment:
I would have guessed the initial commit wasn't needed. Maybe that is a bug we can fix.

Reply:
Agreed, I was surprised as well. I was getting "cannot find branch master" (or prefix-master) without it. I opened an issue here https://github.com/cardstack/cardstack/issues/832 and will add a note that we need to come back and adjust this part and our git guide when we fix it.